nrf_security/README.rst

```diff
@@ -8,8 +8,8 @@ The Nordic Security Module (nrf_security) provides an integration between Mbed T
 This module includes an Mbed TLS glue layer to enable both hardware-accelerated and software-based Mbed TLS implementation at the same time.
 
 .. note::
```

Comment:
Does this conform to the `note` guideline:
> Note - should contain only essential information that must catch the users' attention even if they weren't reading anything else.

Reply:
Removed note.